shared/src/proxies/documents/addPaperFilingProxy.js

```diff
@@ -1,26 +1,31 @@
 const { post } = require('../requests');
 
 /**
- * addPaperFilingProxy
+ * editPaperFilingProxy
```

Comment:
hm this doesn't actually reflect the function name.

Reply:
Updated!